lib/PhantomBrowser.js

```diff
@@ -25,7 +25,11 @@ PhantomBrowser.prototype.__proto__ = EventEmitter.prototype;
 PhantomBrowser.prototype.start = function() {
     var self = this;
 
-    var phantomjs = require('phantomjs');
+    try {
+        var phantomjs = require('phantomjs-prebuilt');
+    } catch (err) {
+        var phantomjs = require('phantomjs');
```

Comment:
I was trying to make this change backwards-compatible, since zuul users install phantomjs separately. However, since this is a major bump to phantomjs, it's probably better to explicitly avoid backwards compatibility here. Fixed in https://github.com/defunctzombie/zuul/commit/eb2c28547104bad3fd614758f7ff4f6d09ffeb79